bluepyopt/electrical/parameters.py

```diff
@@ -93,6 +93,10 @@ def check_bounds(self):
                     (self.name, self._value, str(self.lower_bound),
                      str(self.upper_bound)))
 
+    def destroy(self):
```

Comment:
Don't agree. If you pass an object to another object that will call 'destroy()' on it, but that object doesn't naturally support it, you're very probably doing something wrong. Python should then throw an exception instead of being silent.